### PR TITLE
move repeated code to components

### PR DESCRIPTION
**Pull request by Dhayanandh14**
move repeated code to components

### CODE REVIEW

File: src/components/Modal.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+           <div className="mb-3">
+           <label id="studentInterviewerFormControlInput">Interviewer Name</label>
+            <Forminput type="text" name="interviewer_name" placeholder="Enter Student Interviewer Name"
+            id="studentInterviewerFormControlInput" onChange={props.onChange}/>
+           </div>
+
+
+           <div className="mb-3">
+            <label for="studentInterviewerReviewFormControlInput">Enter Student Interviewer Review</label>
```
Comment (logesh-freshworks) [inline src/components/Modal.js]: You can add the label as well to your reusable component.

File: src/App.js [hunk 2]
```diff
@@ -1,65 +1,58 @@
...
-
-          {< Route path="/dashboard">
+          <Route path="/dashboard">
             <Dashboard />
-          </Route>}
-
-
-          {<Route path="/students" exact>
+          </Route>
+          <Route path="/students" exact>
```
Comment (logesh-freshworks) [inline src/App.js:24]: You can use prettier for formatting. You can install the prettier plugin in vscode. Also enable format on save in your vscode settings. That way your files will be auto-formatted when ever you save.

File: src/pages/ReportByName.js [hunk 3]
```diff
@@ -1,29 +1,21 @@
...
-import { useState } from "react";
+import React, { useEffect, useState } from "react";
 import SideBarComponent from "../components/SidebarComponent";
-import AttendanceService from "../Services/AttendanceService";
 import ReportService from "../Services/ReportService";
-import UserService from "../Services/UserService";
-import "./Report.css"
+import "./Report.css";
 
 const Reportbyname = () => {
```
Comment (logesh-freshworks) [inline src/pages/ReportByName.js]: Always use pascal casing for component names

File: src/pages/CoachEditInfo.js [hunk 4]
```diff
@@ -1,27 +1,27 @@
...
+    status:"",
+    blood_group:"",
+    communication_address:"",
+    emergency_contact_name:"",
+    emergency_contact_number:"",
+    profile_job:""
+  })
   useEffect(() => {
     CoachService.getCoachInfoById(props.id).then((res) => {
       console.log(res.data)
```
Comment (logesh-freshworks) [inline src/pages/CoachEditInfo.js]: Please remove console.log

**Review by logesh-freshworks — CHANGES_REQUESTED** [review]

File: src/pages/CoachEditInfo.js (hunk 4 above)
Comment (Dhayanandh14) [reply src/pages/CoachEditInfo.js]: removed all console.log

File: src/pages/ReportByName.js (hunk 3 above)
Comment (Dhayanandh14) [reply src/pages/ReportByName.js]: updated component names

File: src/App.js (hunk 2 above)
Comment (Dhayanandh14) [reply src/App.js:24]: formatted all files

File: src/components/Modal.js (hunk 1 above)
Comment (Dhayanandh14) [reply src/components/Modal.js]: label component created

**Review by logesh-freshworks — APPROVED** [review]

**Pull request merged**